diff --git a/node_modules/tsd/dist/cli.js b/node_modules/tsd/dist/cli.js
index 5769b7f..34dae92 100755
--- a/node_modules/tsd/dist/cli.js
+++ b/node_modules/tsd/dist/cli.js
@@ -29,8 +29,7 @@ const cli = meow(`
 (() => __awaiter(void 0, void 0, void 0, function* () {
     updateNotifier({ pkg: cli.pkg }).notify();
     try {
-        const options = cli.input.length > 0 ? { cwd: cli.input[0] } : undefined;
-        const diagnostics = yield lib_1.default(options);
+        const diagnostics = yield lib_1.default({cwd:process.cwd(),typingsFile: 'test-d'});
         if (diagnostics.length > 0) {
             throw new Error(formatter_1.default(diagnostics));
         }
